app/docker/docker-compose.kafkaview.yml

```diff
@@ -11,6 +11,8 @@ services:
     depends_on:
       - kafka
     restart: on-failure
+    ports:
+      - "${DAB_APPS_KAFKAVIEW_PORT:-6080}:6080"
```

Comment:
For routable lab networks you will also want to `expose` that port.

Reply:
Done